pkg/controllers/podgroup_controller.go

```diff
@@ -109,6 +111,11 @@ func (r *PodGroupReconciler) Reconcile(ctx context.Context, req ctrl.Request) (c
 		if len(pods) >= int(pg.Spec.MinMember) {
 			pgCopy.Status.Phase = schedv1alpha1.PodGroupPreScheduling
 			fillOccupiedObj(pgCopy, &pods[0])
+		} else {
```

Comment:
why add this else? Is this necessary to remove PostBind?

Reply:
Yes, this else is unnecessary.
`PostBind` logic has been implmented in PodGroup controller. We can delete it as stated in comments.
https://github.com/kubernetes-sigs/scheduler-plugins/blob/2ea5a971705f0ba21052be7edff0f76323190ed7/pkg/coscheduling/core/core.go#L207-L209

Reply:
👍